core/tests/test_ryuk.py

```diff
@@ -1,9 +1,14 @@
+import pytest
+
+from contextlib import contextmanager
+
 from testcontainers.core import container
 from testcontainers.core.container import Reaper
 from testcontainers.core.container import DockerContainer
 from testcontainers.core.waiting_utils import wait_for_logs
 
 
+@pytest.mark.skip("invalid test - ryuk logs 'Removed' right before exiting")
```

Comment:
Why don't we remove the test if it is invalid?
I think the test should check, that the Ryuk container and the Docker container are gone in this case?